### ISSUE TITLE
Missing License

### ISSUE THREAD

**Issue by BoBiene** [wiesson/DDEAgent#2]
Hi, 

im wondering under what license i can use your code.

Thanks for clarifying!

regards,
Bo Biene

**Comment by wiesson** [comment]
That's true. The codebase is quite old and not finished. The adapter from systeminsights for the Sinumerik 840d is missing as well. I can add a MIT licence if it helps you!

**Comment by BoBiene** [comment]
MIT license would be great, thank you!
